specification/src/chapter2.adoc

```diff
@@ -816,11 +903,10 @@ privileged software.
 | Requirement
 
 | CAT_NNN
-| Valid local sealing keys SHOULD only be generated in secured state.
+| Local sealed assets MUST only be possible to unseal on the same physical instance of a system that they were sealed on.
 
-| CAT_NNN
-| Valid local sealing keys MAY be generated in a trusted debug state for
-unaffected software components.
 |===
 
+For example, using sealing keys derived from a hardware provisioned _hardware unique key (HUK)_.
```

Comment:
```suggestion
For example, using sealing keys derived from a silicon-vendor provisioned _hardware unique key (HUK)_.
```

Reply:
"Hardware provisioned assets" are used throughout this document. This specification does not define a specific provisioning model. Depending on use case they can be provisioned at any stage in a supply chain, not necessarily by a silicon vendor.

Reply:
"hardware provisioned" sounds odd per se - since HW does not provision itself - I agree that the provisioning is done either by the silicon manufacturer, or some entity downstream in the supply chain - hence was suggesting some specific actor that does the provisioning. 

we could state it as : For example, using sealing keys derived from a silicon-vendor-(or other downstream entity)-provisioned _hardware unique key (HUK)_.

Reply:
I have used "hardware provisioned assets" throughout the document. Just realised it is not formally defined anywhere. Added a formal definition in the "Assets" section in the reference model (chapter 2) ("immutable asset provisioned in hardware by...").